docs/router/CurveRouterNG.md

```diff
@@ -130,36 +136,118 @@ The contract utilizes interfaces for all relevant Curve pools, such as StableSwa
+
+The array structure includes the following elements: **`[i, j, swap_type, pool_type, n_coins]`**. The first array of this multidimensional setup corresponds to the initial token exchange in the swap sequence.
+
+**`i`**: Index of the input token, fetched from `pool.coins('token address')`.
+
+**`j`**: Index of the output token, fetched from `pool.coins('token address')`.
+
+| `swap_type` | Description |
+| :---: | ----------- |
+| 1 | Standard token `exchange` |
+| 2 | Underlying token `exchange_underlying` |
+| 3 | Underlying exchange via zap for factory stable metapools and crypto-meta pools (`exchange_underlying` for stable, `exchange` in zap for crypto) |
+| 4 | Coin to LP token "exchange" (effectively `add_liquidity`) |
+| 5 | Lending pool underlying coin to LP token "exchange" (effectively `add_liquidity`) |
+| 6 | LP token to coin "exchange" (effectively `remove_liquidity_one_coin`) |
+| 7 | LP token to lending or fake pool underlying coin "exchange" (effectively `remove_liquidity_one_coin`) |
+| 8 | Specialized swaps like ETH <-> WETH, ETH -> stETH or frxETH, and cross-liquidity between staked tokens (e.g., stETH <-> wstETH, frxETH <-> sfrxETH) |
+| 9 | SNX-related swaps (e.g., sUSD, sEUR, sETH, sBTC) |
+
+| `pool_type` | Description |
+| :---------: | ----------- |
+| 1 | Stable pools using the Stableswap algorithm |
+| 2 | Two-coin Crypto pools using the Cryptoswap algorithm |
+| 3 | Tricrypto pools with three coins using the Cryptoswap algorithm |
+| 4 | Llamma pools, typically used in crvUSD and lending markets |
+
+**`n_coins`**: Number of coins contained within the pool.
+
+
+=== "Example"
+
+    ```py
+    [
+        [2, 0, 1, 3, 3],    # first swap:  wETH -> USDC using tricryptoUSDC pool
+        [0, 1, 1, 1, 2],    # second swap: USDC -> crvUSD using crvUSD/USDC pool
+        [0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0],
+    ]
+    ```
+
+*Let's take a closer look at the first array, which represents the swap parameters for the first exchange (wETH -> USDC). `i = 2` because the coin index value of wETH in the tricryptoUSDC pool is 2. This can be obtained by calling `tricryptoUSDC.coins(n)`. Similarly, `j = 0` because USDC has the coin index value of 0. The swap type is a regular exchange, represented by `0`. The `pool_type` is 3, as it is a tricrypto pool (a cryptoswap algorithm consisting of three coins: USDC, wBTC, and wETH). The last value in the array represents the number of coins in the pool, which is 3.*
+
+*The values of the second array should be set according to the crvUSD/USDC pool.*
+
+
+---
+
+
+## **Exchanging Tokens**
+
+The router has a single `exchange` function, which allows up to 5 swaps in a single transaction. 
+
+Routing and swap parameters need to be determined off-chain. The exchange functionality of the router is designed for gas efficiency over ease-of-use. A according JavaScript library can be found on [:material-github: GitHub](https://github.com/curvefi/curve-router-js).
```

Comment:
> A according JavaScript library

Something not quite right here, perhaps should be "An accompanying JavaScript library" ?

Could also add that the library is used for determining the offchain swap parameters, just for clarity.